docs/tutorials/training/saving-loading.md

```diff
@@ -0,0 +1,205 @@
+# Saving and Loading TensorFlow.js models
+
+TensorFlow.js provides functionality for saving and loading models that have been created with
+the [`layers`](https://js.tensorflow.org/api/0.14.2/#Models) API. These may be models you have trained yourself or those trained by others. A key benefit of using the
+layers api is that the models created with it are serializable and this is what we will explore in this tutorial.
+
+This tutorial will focus on saving and loading TensorFlow.js models (identifiable by JSON files). We can also import TensorFlow Python models.
+Loading these models are covered in the following two tutorials:
+
+  - [TODO] TK add link to Keras Model Importing tutorial
+  - [TODO] TK add link to Python Graphdef Model Importing tutorial
+
+
+## Saving a tf.Model
+
+[`tf.Model`](https://js.tensorflow.org/api/0.14.2/#class:Model) and [`tf.Sequential`](https://js.tensorflow.org/api/0.14.2/#class:Model)
+both provide a function [`model.save`](https://js.tensorflow.org/api/0.14.2/#tf.Model.save) that allow you to save the
+_topology_ and _weights_ of a model.
+
+> Topology: This is a file describing the architecture of a model (i.e. what operations it uses). It contains references to the models's weights which are stored externally.
+
+> Weights: These are binaray files that store the weights of a given model in an efficient format. They are generally stored in the same folder as the topology.
+
+Let's take a look at what the code for saving a model looks like
+
+```js
+const saveResult = await model.save('localstorage://my-model-1');
+```
+
+A few things to note:
+
+- The `save` method takes a URL-like string argument that starts with a **scheme**. This describes the type of destination we are trying to save a model to. In the example above the scheme is `localstorage://`
+- The scheme is followed by a **path**. In the example above the path is `my-model-1`.
+- The `save` method is asynchronous.
+- The return value of `model.save` is a JSON object that carries information such as the byte sizes of the model's topology and weights.
+- The environment used to save the model does not impact which environments can load the model. Saving a model in node.js does not prevent it from being loaded in the browser.
+
+Below we will examine the different schemes available.
+
+### Local Storage (Browser only)
+
+**Scheme:** `localstorage://`
+
+```js
+model.save('localstorage://my-model');
```

Comment:
is model.save() async? If yes, might be good to say `const res = await model.save()` everywhere consistently.

Reply:
i'm generally dropping the result part since we aren't doing any thing with it. But await would be good to add.